symfony.lock

```diff
@@ -44,6 +44,9 @@
     "doctrine/deprecations": {
         "version": "v0.5.3"
     },
+    "doctrine/deprecations": {
+        "version": "v0.5.3"
+    },
```

Comment:
Should be reverted because the block is redundant now: see the three lines above it.

Reply:
Done in https://github.com/symfony/demo/pull/1277/commits/8c4defe7556f4a08cf95d2a5d1ad977a5c2c47a2. Thanks!